examples/task.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: test-app
```

Comment:
test-task ?

Reply:
Yes, it's the name of the task. Any issue?

Reply:
I meant to rename task, to `test-task`, so it aligned with `test-pipeline` name